app/js/redux-store.jsx

```diff
@@ -0,0 +1,41 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+import {
+  createStore, applyMiddleware, combineReducers, compose,
+} from 'redux';
+import promiseMiddleware from 'redux-promise-middleware';
+import createSagaMiddleware from 'redux-saga';
+import thunkMiddleware from 'redux-thunk';
+import { sagas as openmrsSagas, reducers as openmrsReducers } from '@openmrs/react-components';
+import { reducer as reduxFormReducer } from 'redux-form';
+
+import * as reducers from './reducers';
+
+const sagaMiddleware = createSagaMiddleware();
+
+const middlewares = [sagaMiddleware, thunkMiddleware, promiseMiddleware()];
```

Comment:
I'm wondering if we want to standardize on using sagas instead of thunks for async events?  I know we use Thunks in the Order Entry OWA, but then I decided to use Sagas in the react-components, so we need to at least use Sagas here.  Can we use Sagas for everything, or do we think there are use cases that Thunks are better suited for?